schemas/container.json

```diff
@@ -173,6 +173,19 @@
         "additionalProperties": false
       }
     },
+    "company": {
+      "type": "object",
+      "properties": {
+        "orgId": {
+          "type": "string",
+          "minLength": 1,
+          "pattern": "[A-Z0-9]{24}@AdobeOrg"
+        }
+      },
+      "required": [
+        "orgId"
+      ]
+    },
```

Comment:
Let's add `"additionalProperties": false` to the `company` object so we catch superfluous data.